src/index.html

```diff
@@ -7,8 +7,65 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class = "header">
+      <a href = "index.html" class = "logo-link">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+          class = "logo"
+        >
+      </a>
+
+      <nav class = "nav">
+        <ul class = "nav-list">
+          <li class = "nav-item">
+            <a href = "index.html" class = "nav-link is-active">Apple</a>
```

Comment:
1. following BEM recommendations for naming classes
2. link shouldn't lead to home page
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
```